lib/http.js

```diff
@@ -189,7 +190,6 @@ export async function addRoutes({app, service} = {}) {
           }
           ({uses: challengeUses} = result);
         }
-
```

Comment:
We could probably revert these whitespace changes as well so we don't have any changes here. In the end, it looks like the only changes might be in the test suite and package.json.

Reply:
Reverted.